### DISCUSSION TITLE
Metrics endpoints not visible in Postman collection?

### DISCUSSION THREAD

**Discussion by psascue** [aws-solutions-library-samples/guidance-for-aws-sustainability-insights-framework#13] [Q&A] [answered]
Hi, another noob question here.

According to [this section](https://github.com/aws-solutions-library-samples/guidance-for-aws-sustainability-insights-framework/blob/main/docs/walkthrough.md#1-creating-metric-kpi), I can do a POST to create a metric.

I prefer using Postman as much as possible and I used it to do the earlier task of creating and listing the organization structure. I looked for the metrics endpoint in Postman but is it not there?
![image](https://github.com/aws-solutions-library-samples/guidance-for-aws-sustainability-insights-framework/assets/84527760/1f90a9cd-c35c-4141-8262-5b7cf615e832)

To test if I broke something, I used curl and the POST succeeded. I used Postman for earlier steps, so I was expecting to be able do the same for metrics. Did I miss anything?

**Comment by williamsia** [comment, marked as answer]
Hi @psascue , it looks like we haven't updated the endpoint with some our newest resource endpoint ( such as metric ) , I'll raise a ticket internally to update this. In the meantime look at this openapi/swagger definition [here](https://github.com/aws-solutions-library-samples/guidance-for-aws-sustainability-insights-framework/blob/main/typescript/packages/apps/pipelines/docs/swagger.json) to look at the structure of the payload and response. 

Thanks again for reporting this.

**Reply by psascue** [reply]
Thanks for confirming @williamsia !

**Reply by arnefromsa** [reply]
Thanks for the feedback on this. Is there an issue raised on this project, or is this internal @williamsia  ?
